nucypher/network/nodes.py

```diff
@@ -927,11 +927,17 @@ def learn_from_teacher_node(self, eager=False, canceller=None):
             # except sprout.Invalidsprout:
             #     self.log.warn(sprout.invalid_metadata_message.format(sprout))
 
-            except sprout.SuspiciousActivity:
```

Comment:
Try simple adding the import `from nucypher.network.protocols import SuspiciousActivity`

and then changing this line to be `except SuspiciousActivity`.

Reply:
i believe i have fixed this in the (latest) commit 274d3c410edbf5a9b37531c991d0cbcdc5f8be0c